src/SdkResolver/SdkResolver.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(CoreSdkTargetFramework)</TargetFramework>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <GenerateDependencyFile>false</GenerateDependencyFile>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <GenerateRuntimeCofigurationFiles>false</GenerateRuntimeCofigurationFiles>
+    <AssemblyName>UNUSED</AssemblyName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.MSBuildSdkResolver" Version="$(MicrosoftDotNetMSBuildSdkResolverPackageVersion)" ExcludeAssets="all" GeneratePathProperty="true"/>
+    <PackageReference Include="runtime.win-x86.Microsoft.NETCore.DotNetHostResolver" Version="$(MicrosoftNETCoreDotNetHostResolverPackageVersion)" ExcludeAssets="all" GeneratePathProperty="true" />
+    <PackageReference Include="runtime.win-x64.Microsoft.NETCore.DotNetHostResolver" Version="$(MicrosoftNETCoreDotNetHostResolverPackageVersion)" ExcludeAssets="all" GeneratePathProperty="true" />
+  </ItemGroup>
+
+  <Target
+    Name="GenerateLayout" 
+    Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x86' "
+    AfterTargets="Build"
```

Comment:
Can you do `BeforeTargets="AfterBuild"` instead? That's pretty closely equivalent and avoids some corner cases with dependencies.

Reply:
Will do. This is a simplified version of redist, which is also AfterTargets="Build". I'll change that too.

Incidentally, I refrained from cleaning up a bunch of related things. I think we should have a pattern for layout projects and split off a bunch of them from monolithic redist.csproj that currently MSBuilds DownloadPackage.csproj with differerent global properties. This should allow everything to be pulled down from initial restore and not require nested restores. The binlog will be easier to parse, and the buil will be faster. I'll file a separate issue for that.